autosklearn/estimators.py

```diff
@@ -232,8 +232,13 @@ def __init__(
 
             Not all keys returned by scikit-learn are supported yet.
 
+        performance_over_time\_ : pandas.core.frame.DataFrame
+            A ``DataFrame`` containing the models performance over time data. Can be
+            used for plotting directly. Please refer to the example
+            `Train and Test inputs <examples/40_advanced/example_pandas_train_test.html>`_.
```

Comment:
Found the issue! Changed to :ref:`Train and Test Inputs <sphx_glr_examples_40_advanced_example_pandas_train_test.py>`